src/utils.ts

```diff
@@ -22,4 +22,28 @@ const uploadImage = async (imageB64: string): Promise<string> => {
   return responseData.data;
 };
 
-export default uploadImage;
+/**
+ * Remove an image from AppDev servers
+ *
+ * @param imageURL the image URL to remove
+ * @returns true if the deletion was successful; otherwise false
+ */
+const removeImage = async (imageURL: string): Promise<boolean> => {
+  // Delete image via a POST request
+  const payload = {
+    bucket: UPLOAD_BUCKET,
```

Comment:
Once again would recommend moving this upload bucket variable into the env file as well and purging from git history